webui/settings.py

```diff
@@ -259,8 +259,16 @@ def has(setting: str) -> bool:
             return False
 
     @staticmethod
-    def getBoolean(config_path: str, default=False) -> bool:
-        value = ConfigFile().get(config_path, default)
+    def getBoolean(config_path: str, namespace: str = None, default=False) -> bool:
```

Comment:
I wonder, why we do not change `get_feature_property()` so it handle bool types as well.

And then drop this `getBoolean()`.

Reply:
ok

Reply:
after spending another half-day on this PR I realized that this is enough :) while generally agree with you I would like to move this into separate PR . I create issue to not forget about this https://github.com/SUSE/pcw/issues/108